src/python/grapl_analyzerlib/grapl_analyzerlib/nodes/base.py

```diff
@@ -0,0 +1,212 @@
+import json
+from typing import Any, TypeVar, Set, Type, Optional
+
+from grapl_analyzerlib.grapl_client import GraphClient
+from grapl_analyzerlib.node_types import (
+    PropType,
+    PropPrimitive,
+)
+from grapl_analyzerlib.queryable import Queryable
+from grapl_analyzerlib.schema import Schema
+from grapl_analyzerlib.viewable import Viewable
+
+BCQ = TypeVar("BCQ", bound="BaseQuery")
```

Comment:
what's the C in these

Reply:
Definitely a copy paste failure good catch.